thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionJobResource.java

```diff
@@ -307,14 +308,51 @@ public Response tuneAlertFilter(@PathParam("function_id") String id,
       if (alertFilterAutotune.isUpdated()){
         anomalyFunctionSpec.setAlertFilter(tunedAlertFilter);
         anomalyFunctionDAO.update(anomalyFunctionSpec);
-        alertFilter.setParameters(tunedAlertFilter);
         LOG.info("Model has been updated");
       } else {
         LOG.info("Model hasn't been updated because tuned model cannot beat original model");
       }
     } catch (Exception e) {
       LOG.warn("AutoTune throws exception due to: {}", e.getMessage());
     }
-    return Response.ok(alertFilter.toString()).build();
+    return Response.ok(alertFilterAutotune.isUpdated()).build();
+  }
+
+  /**
+   * The endpoint to evaluate alert filter
+   * @param id: function ID
+   * @param startTime: startTime of merged anomaly
+   * @param endTime: endTime of merged anomaly
+   * @return feedback summary and precision/recall as json object
+   * @throws Exception when data has no positive label or model has no positive prediction
+   */
+  @POST
+  @Path("/eval/filter/{function_id}")
+  public Response evaluateAlertFilterByFunctionId(@PathParam("function_id") String id,
+      @QueryParam("startTime") Long startTime,
+      @QueryParam("endTime") Long endTime) throws Exception {
```

Comment:
please be consistent with parameter names. Keep camel case for functionId also keep variable name same as parameter name.